### PR TITLE
hwopus: Implement OpenHardwareOpusDecoderForMultiStreamEx and DecodeInterleavedForMultiStream

### PR DESCRIPTION
**Pull request by FearlessTobi**
Allows MLB The Show 22 to boot.

Fixes https://github.com/yuzu-emu/yuzu/issues/7911.

### CODE REVIEW

File: src/core/hle/service/audio/hwopus.h [hunk 1]
```diff
@@ -18,7 +18,7 @@ struct OpusMultiStreamParametersEx {
     u32 number_stereo_streams;
     u32 use_large_frame_size;
     u32 padding;
-    std::array<u32, 64> channel_mappings;
+    std::array<u8, 0x100> channel_mappings;
 };
```
Comment (german77) [inline src/core/hle/service/audio/hwopus.h:22]: This is `nn::codec::detail::HardwareOpusMultiStreamDecoderParameterInternalEx` also add an assert on the struct size.

**Comment by Moonlacer** [comment]
Does this affect MLB the Show 23 as well? I don't have the game, but I know it exists

**Comment by german77** [comment]
Needs someone to test. We don't know if other games use the same features.

File: src/core/hle/service/audio/hwopus.h (hunk 1 above)
Comment (TheGrimReaper798) [reply src/core/hle/service/audio/hwopus.h:22]: mlb the show 23 works but there are vertex explosions on nvidia cards

**Comment by TheGrimReaper798** [comment]
mlb the show 23 works but there are vertex explosions on nvidia cards

**Review by liamwhite — APPROVED** [review]

**Pull request merged**